lib/modules/apostrophe-ui/views/components/fields.html

```diff
@@ -13,11 +13,13 @@
   </label>
 {%- endmacro %}
 
-{% macro select(name, options) -%}
+{# "selected" value is typically not passed, but we do push it in when rendering manage filters. -Tom #}
```

Comment:
Would prefer formal documentation of the macro with information about all parameters.